### ISSUE TITLE
Ideal state: reading from readBin of connections to flow control of number of lines

### ISSUE THREAD

**Issue by groliver-pcci** [BioStatMatt/sas7bdat#13]
The current function is set to read from buffer, but when the SAS files are extremely large this chokes.

My thought is to fork a set of the read functions using "rb" file connection, parsing the header

**Comment by BioStatMatt** [comment]
Yes, this is a weakness. However, this code is mostly intended as a demonstration implementation of the file format reader. Unfortunately, this code is not actively maintained.
